tincr/io/library/genFamilyInfo.tcl

```diff
@@ -176,9 +179,10 @@ proc processSite {s type compatible is_alt fo} {
 		
 		#print the bel information 
 		set tmpname [suffix $b "/"]
+		set tmptype [prefix $b "/"]
```

Comment:
Did you test this code? Because I'm pretty sure it doesn't work. Say we have a bel that is called `SLICE_X3Y95/D6LUT`. Your prefix function will return "SLICE_X3Y95" which is certainly not the type. You should be using the `TYPE` property on bel objects